app/components/groups_list/group_list_tree_component.html.erb

```diff
@@ -0,0 +1,12 @@
+<ul class="groups-list group-list-tree flex flex-col">
+  <%= render GroupsList::GroupRowComponent.with_collection(groups.take(limit), path: path, path_args: path_args, collapsed: collapsed) %>
+  <% if show_more %>
+    <li class="group-row">
+      <p class="py-3 pr-3 flex items-center">
+        <%= link_to "View all #{groups.count} subgroups",
```

Comment:
**nitpick (blocking):** Can this text be translated, also this link does not work. Can you link this to the group page which doesn't show the subgroups but is where they will be shown. Also maybe this is where you intended to use the svg that I commented on above.

The other option is in this PR we show all the children and leave the `View more` to a followup PR.

Reply:
I removed it in 7e0d33d Agree we can do this on a subsequent PR